### PR TITLE
[AARCH64 Automation] Incorporate new aarch64 SUT machine into grub boot method

### PR DESCRIPTION
**Pull request by waynechen55**
New aarch64 SUT machine in different domain needs the same treatment as other ones

- Related ticket: AARCH64 Automation
- Needles: n/a
- Verification run: https://10.67.20.171/tests/256

### CODE REVIEW

File: tests/boot/boot_from_pxe.pm [hunk 1]
```diff
@@ -64,7 +64,7 @@ sub run {
         my $repo        = $openqa_url . "/assets/repo/${image_name}";
         my $sut_machine = get_var('SUT_IP', 'nosutip');
         my $key_used    = '';
-        if (is_remote_backend && check_var('ARCH', 'aarch64') && ($sut_machine =~ /arch\.suse\.de/img)) {
+        if (is_remote_backend && check_var('ARCH', 'aarch64') && ($sut_machine =~ /(arch\.suse\.de|qa2\.suse\.asia)$/img)) {
```
Comment (foursixnine) [inline tests/boot/boot_from_pxe.pm]: Please import `is_aarch64` and use it :) 

Also you could make the sentence more legible by using a `qr//`
```suggestion
        my $allowed_sut_re = qr/(arch\.suse\.de|qa2\.suse\.asia)/im;
        if (is_remote_backend && is_aarch64 && ($sut_machine =~ $allowed_sut_re)) {
```

Comment (waynechen55) [reply tests/boot/boot_from_pxe.pm]: Done

**Comment by foursixnine** [comment]
@waynechen55 You forgot to tidy up :)

**Comment by waynechen55** [comment]
> @waynechen55 You forgot to tidy up :)

Done

**Pull request merged**